client/pnl.go

```diff
@@ -99,6 +95,9 @@ func (c *Client) PNL(ctx context.Context) (PNLResponse, error) {
 		}
 
 		return PNLResponse{}, fmt.Errorf("status: %d\nerror: %s", resp.StatusCode, pnlResponse.Message)
+	
+	} else if err != nil {
```

Comment:
why was this change made?

Reply:
This will correctly handle all the errors i.e 4xx, 5xx or json decoding error in the correct order